jest.config.js

```diff
@@ -1,6 +1,19 @@
-const { defaults } = require('jest-config');
-
 module.exports = {
+  verbose: true,
+  testEnvironment: 'jsdom',
   moduleDirectories: ['node_modules', 'src'],
-  testEnvironment: "jsdom"
+  roots: ['<rootDir>/src'],
+  transform: {
+    '^.+\\.[t|j]sx?$': 'ts-jest',
+    '^.+\\.scss$': 'jest-scss-transform'
```

Comment:
change jest-scss-transform to [`jest-css-modules-transform`](https://www.npmjs.com/package/jest-css-modules-transform)